mixin/README.md

```diff
@@ -100,7 +100,7 @@ This project is intended to be used as a library. You can extend and customize d
   dashboard+:: {
     prefix: 'Thanos / ',
     tags: ['thanos-mixin'],
-    namespaceQuery: 'kube_pod_info',
+    namespaceMetric: 'kube_pod_info',
```

Comment:
This could be a fully-fledged query, thus the name.

Reply:
Currently in the grafana builder this is just defined as a metric name, so if we specify `namespaceQuery: 'kube_pod_info{app="..."}'` it will rather appear as `kube_pod_info\{app=\"...\"\}{}` in the json which makes it invalid.

https://github.com/thanos-io/thanos/blob/master/mixin/dashboards/compact.libsonnet#L133